### DISCUSSION TITLE
Issue about new propeller: Propeller 47-17

### DISCUSSION THREAD

**Discussion by PimKanjana** [bitcraze/discussions#1114] [Q&A]
I have tested the new propellers 47-17 using Crazyflie Client several times. we tested the autonomous flight using Lighthouse. The Crazyflie was unstable with the new propellers and always crashed after flying for a few minutes (usually while I was about to land it). After I change back to the old-type propellers, everything works fine. 

![S__82059268](https://github.com/bitcraze/discussions/assets/43192896/af47e752-219a-42bb-881b-e723b340db65)

Btw, every time I tested the propellers, it's always test failed!!! 

![S__82067461](https://github.com/bitcraze/discussions/assets/43192896/63dfa584-a2c7-4ec2-b9b2-22e9f4302e3a)

**Comment by knmcguire** [comment]
Hi! 

That is indeed a bit of a problem. Since the crazyflie can fly at least for a while, it feels to me that the PID controller needs to be tuned a bit probably. Have you found the [tuning tab](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/tuning_tab/) yet? Since the propellers are more efficient I would say that the gains should be tuned down a bit.

I'm not surprised that the prop test fails since that has been calibrated on the standard propellers. 

This is a better question for @tobbeanton but he is on holiday still now. I'll notify him on this once he is back and perhaps he has some more tips.

**Reply by PimKanjana** [reply]
Thank you so much for your quick response. I’ll try it!

**Reply by tobbeanton** [reply]
Like knmcguire said the propeller test is not tuned for the new propeller. That said the motor M1 seam to have a lot of vibrations. Could that prop be damaged/unbalanced? Do you have another one to try with instead?
